ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/AuthHandlers.java

```diff
@@ -52,50 +57,72 @@ static void setupAuthHandlers(final Server server, final Router router,
 
     systemAuthenticationHandler.ifPresent(handler -> router.route().handler(handler));
 
-    if (jaasAuthHandler.isPresent() || authenticationPlugin.isPresent()) {
-      router.route().handler(AuthHandlers::pauseHandler);
+    if (jaas.isPresent() || authenticationPlugin.isPresent()) {
```

Comment:
these changes are not matching with 7.3.x.

Reply:
Thanks for pointing this out. I see there was one more commit done post the main changes. I have added that commit as well(https://github.com/confluentinc/ksql/pull/9446)